test/gui/shared/scripts/helpers/ConfigHelper.py

```diff
@@ -73,12 +73,14 @@ def get_default_home_dir():
     'clientConfigDir': 'CLIENT_CONFIG_DIR',
     'guiTestReportDir': 'GUI_TEST_REPORT_DIR',
     'ocis': 'OCIS',
-    'screenRecordOnFailure': 'SCREEN_RECORD_ON_FAILURE',
+    'recordVideoOnFailure': 'RECORD_VIDEO_ON_FAILURE',
```

Comment:
how about writing these attributes in snake_case?

Reply:
would be great. but not in this PR.

Reply:
Remaining attributes can be renamed in the future PRs.